tests/acceptance/diagrams/activity-diagram/draw-objects-over-others-test.js

```diff
@@ -42,7 +42,7 @@ function createData(systemName, diagramName, elementType, elementAttributes) {
           elementType,
           { x: 300, y: 300 },
           { width: 40, height: 40 },
-          elementAttributes,
+          elementAttributes
```

Comment:
```suggestion
          elementAttributes,
```
Висящие запятые наоборот надо везде ставить